src/core/mem.c

```diff
@@ -264,19 +269,21 @@ bool mem_vm_img_in_phys_rgn(struct vm_config* vm_config) {
     return img_in_rgn;
 }
 
-bool mem_reserve_physical_memory(struct page_pool *pool)
+bool mem_reserve_physical_memory(struct page_pool* pool)
 {
-    if (pool == NULL) return false;
+    if (pool == NULL) {
+        return false;
+    }
 
     for (size_t i = 0; i < config.vmlist_size; i++) {
-        struct vm_config *vm_cfg = &config.vmlist[i];
+        struct vm_config* vm_cfg = &config.vmlist[i];
         size_t n_pg = NUM_PAGES(vm_cfg->image.size);
         struct ppages ppages = mem_ppages_get(vm_cfg->image.load_addr, n_pg);
 
         // If the vm image is part of a statically allocated region of the same
         // vm, we defer the reservation of this memory to when we reserve the
-        // physical region below. Note that this not allow partial overlaps. If the
-        // image must be entirely inside a statically allocated region, or
+        // physical region below. Note that this not allow partial overlaps. If
+        // the image must be entirely inside a statically allocated region, or
         // completely outside of it. This avoid overcamplicating the reservation
         // logic while still covering all the useful use cases.
```

Comment:
After this PR is accepted we need to rewrap comment lines to 100ch limit.

Reply:
We can do it in this PR.

Reply:
Wrapped all comments to 100 in commit [59fecb5](https://github.com/bao-project/bao-hypervisor/pull/103/commits/59fecb5de26527b45a54f99ad9dd530dd8336c90)

Reply:
thanks!